internal/awstesting/unit/unit.go

```diff
@@ -16,6 +16,7 @@ func init() {
 			Source: "unit test credentials",
 		},
 	}
+	config.Retryer = aws.NewDefaultRetryer()
```

Comment:
Probably want to leave this unset, so that the behavior doesn't change between a config not specifying a retryer an the client automatically creating the default behavior.